pal/src/misc/random.cpp

```diff
@@ -18,34 +18,49 @@ Module Name:
 --*/
 
 #include "pal/palinternal.h"
-#include <sodium.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+#include <mutex>
 
-/*++
-Function :
-RANDOMInitialize
-
-Initialize libsodium
 
-(no parameters)
+#define RANDOM_CACHE_SIZE 8
+constexpr int rand_byte_len = sizeof(unsigned int) * RANDOM_CACHE_SIZE;
```

Comment:
Kind of :smile: - the PAL is currently used only on platforms supporting C++11 and above. Changes to actual ChakraCore code still needs to support VS2013's version of VC++. 
TLDR; in this case it's fine